### DISCUSSION TITLE
Is this Project dead?

### DISCUSSION THREAD

**Discussion by Alfagun74** [sampotts/plyr#2335] [General]
Can someone with power please take care of the dozens of pull requests that fix important bugs?
I love this project but it feels like it is dying. :(

**Comment by sampotts** [comment]
As on Twitter, not dead. I just need to find some time which has been hard in between my full time job at Amazon, my family and trying to buy a house. I’ll make sure to spend some time on it this week to go through PRs etc.

**Reply by Alfagun74** [reply]
tysm ❤

**Reply by GoodnessEzeh** [reply]
Thank you for all your sacrifices, Sampotts.
